src/screens/Login.js

```diff
@@ -118,39 +134,65 @@ class Login extends Component {
   render() {
     const { error } = this.state;
     return (
-      <KeyboardAvoidingView behavior="padding" style={styles.container}>
-        <View style={styles.backgroundStyle}>
-          <View style={styles.viewStyle}>
-            <OpacityButton text="Play without logging!" onPress={this.onAnonymous} />
-            <Text style={styles.signUpText}>
-              Sign up/in
-            </Text>
-            <Text style={styles.infoText}>
-              After logging you would see history of your plays
-            </Text>
-            <StatusBar barStyle="light-content" />
+      <ImageBackground
+        style={styles.backgroundImage}
+        source={require('./../../assets/background.png')}
+      >
+        <KeyboardAvoidingView behavior="padding" style={[styles.view, styles.keyboardAvoidingView]}>
+          <View style={[styles.view, styles.logoView]}>
```

Comment:
I belive all those views could be extracted to seperate functions, this render functions gets a little bit too long.